<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdquisicionDatos</class>
 <widget class="QMainWindow" name="AdquisicionDatos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AdquisicionDatos</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(198, 198, 198);</string>
       </property>
       <property name="text">
        <string>QNX1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(198, 198, 198);</string>
       </property>
       <property name="text">
        <string>QNX2</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(198, 198, 198);</string>
       </property>
       <property name="text">
        <string>QNX3</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(198, 198, 198);</string>
       </property>
       <property name="text">
        <string>QNX4</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(drifscan)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>No. de lineas a leer:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumeroLineasLeerlineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>235</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tiempo de Exposicion (seg):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tiempoExposicionlineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(guiado)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="observarPushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Observar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salirPushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="emcabezadosPushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Emcabezados</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Control de Hardware</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>521</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="modoDeLaLineaDelRelojLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Modo de la Linea del Reloj:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="modoDeLaLineaDelRelojComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="declinacionGradosDecLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Declinacion (grados dec):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="declinacionGradosDecLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="texturaDeLecturaHzLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tasa de Lectura (hz):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="tasaDeLecturaHzLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>526</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Comandos para el Obturador: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="AbrirPushButton">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Abrir</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="CerrarPushButton">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="EstadoObturadorPushButton">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Estado</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Comandos para los Dedos:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="EstadoPushButton">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Estado</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="InicioPushButton">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="SetPushButton">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Posicion para los Dedos (x,y,z,t):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posXLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="posYLineEdit">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>550</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="posZLineEdit">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>550</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="posTLineEdit">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>550</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>531</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Secuencia de Foco:</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tiempo de Exposicion (seg):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Numero de Exposiciones:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tiempoExposicionFocoLineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>15</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="IniciarSec1PushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>11</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Iniciar Sec N0. 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numeroExposicionesLineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="AbortarSecPushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Abortar Secuencia</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Panel de Control de los CCD</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comandoDeObservacionComboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="comandoDeObservacionLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>161</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Comando de Observacion: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="LogTextEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>431</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>550</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Control de Hardware</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>580</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="formatoDeLaImagenLabel">
       <property name="text">
        <string>Formato de la Imagen:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="formatoDeLaImagenComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lineaDeDatosPorArchivoLabel">
       <property name="text">
        <string>Linea de Datos por Archivo:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineaDeDatosPorArchivoLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="indiceDeLaObservacionLabel">
       <property name="text">
        <string>Indice de la Observacion:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="indiceDeLaObservacionLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="modiaPushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modia</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>Panel Administrativo:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Correo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usuariolineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Contraseña:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="IngresarPushButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>97</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingresar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="olvidoPasswordpushButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>120</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 10px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
    color:rgb(0, 0, 116);
    border-style: inset;
}

QPushButton:pressed {
     border-style: inset;
 }

</string>
     </property>
     <property name="text">
      <string>¿Olvido su contraseña?</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionIniciar_Prueba"/>
    <addaction name="actionPanel_Administrativo"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <action name="actionIniciar_Prueba">
   <property name="text">
    <string>Iniciar Prueba</string>
   </property>
  </action>
  <action name="actionCrear_Prueba">
   <property name="text">
    <string>Crear Prueba</string>
   </property>
  </action>
  <action name="actionPanel_Administrativo">
   <property name="text">
    <string>Panel Administrativo</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
